packages/edit-site/src/components/sidebar/global-styles-sidebar.js

```diff
@@ -183,37 +178,48 @@ export default function GlobalStylesSidebar( {
 				</>
 			}
 		>
-			<TabPanel
-				tabs={ [
-					{ name: 'root', title: __( 'Root' ) },
-					{ name: 'block', title: __( 'By Block Type' ) },
-				] }
-			>
-				{ ( tab ) => {
-					/* Per Block Context */
-					if ( 'block' === tab.name ) {
-						return (
-							<GlobalStylesBlockPanels
-								blocks={ blocks }
-								getStyle={ getStyle }
-								setStyle={ setStyle }
-								getSetting={ getSetting }
-								setSetting={ setSetting }
-							/>
-						);
-					}
-					return (
-						<GlobalStylesPanel
-							hasWrapper={ false }
-							context={ root }
+			<Navigation>
+				<NavigationMenu>
+					<GlobalStylesLevel
+						context={ root }
+						getStyle={ getStyle }
+						setStyle={ setStyle }
+						getSetting={ getSetting }
+						setSetting={ setSetting }
+					/>
+					<NavigationItem
+						item="item-blocks"
+						navigateToMenu="blocks"
+						title={ __( 'Blocks' ) }
+					/>
+				</NavigationMenu>
+				<NavigationMenu menu="blocks" parentMenu="root">
+					{ map( blocks, ( _, name ) => (
```

Comment:
Previously we were sorting blocks alphabetically. Should we still do it?

Reply:
Should they follow the inserter's order?

Reply:
Yes, inserter order, and possibly with categories down the road.

Reply:
We'll need a search box too for quick finding.